package.json

```diff
@@ -6,13 +6,13 @@
   "license": "Apache-2.0",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
+    "build": "babel src --out-dir dist/src && babel index.js --out-dir dist/ && cp package.json dist/ && cp LICENSE dist/",
```

Comment:
missing `@babel/cli` dependency.

Reply:
Thanks for help! I have added the dependency and test it with `semantic-release --dry-run`